Crashlytics/Crashlytics/FIRCrashlytics.m

```diff
@@ -307,7 +310,14 @@ - (void)processDidCrashDuringPreviousExecution {
 
 #pragma mark - API: Logging
 - (void)log:(NSString *)msg {
```

Comment:
I want to confirm this remains performant and is necessary for every method (as opposed to just necessary for the setDevelopmentPlatform method).

Can we try running a test app and logging in a loop and confirm the last N logs all make it up? Eg. `for (0...1000) Crashlytics.log(number)`?